docs/dev-guide/contributing/dev_setup.rst

```diff
@@ -10,7 +10,7 @@ For the Impatient
 These instructions will create a developer install of Pulp on a dedicated
 development instance.
 
-* Start a Fedora 20 x86_64 instance that will be dedicated for development with
+* Start a RHEL 7 or Fedora 21/22 x86_64 instance that will be dedicated for development with
```

Comment:
Should this be agnostic about the Fedora version so we don't need to constantly update this. It could refer to them as a `current Fedora` instead maybe?